src/components/counter.ts

```diff
@@ -0,0 +1,31 @@
+import {useState, createElement} from 'react';
+
+const Counter = ({initialValue}:CounterProps)=>{
```

Comment:
**This components (according to the task description) should be a class component.
 Please refactor to follow the requirements.**